### PR TITLE
movies and some styles

### PR DESCRIPTION
**Pull request by Ahmedoudouh**

### CODE REVIEW

File: src/Componnets/header/Header.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import "./Header.css";
 export const Header = () => {
   return (
     <header className="header">
-      <h1 className="title">🎬 Entertainment Hub 🎥</h1>
+      <h1 onClick={()=>window.scroll(0,0)} className="title">🎬 Entertainment Hub 🎥</h1>
```
Comment (younessoudouh) [inline src/Componnets/header/Header.jsx:7]: create function instead!

File: src/footer/button.jsx [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+import React, { useState } from "react";
 
-export const Button = ({ icon , name }) => {
+import "./style.css";
+export const Button = ({ icon, name }) => {
+ const [changeBck,setChangeBck]=useState()
+  function handelclick() {
+    setChangeBck(true)
+  }
+  console.log(changeBck)
```
Comment (younessoudouh) [inline src/footer/button.jsx:9]: remove console

File: src/footer/button.jsx [hunk 3]
```diff
@@ -1,9 +1,14 @@
-import React from "react";
-import "./style.css";
+import React, { useState } from "react";
 
-export const Button = ({ icon , name }) => {
+import "./style.css";
+export const Button = ({ icon, name }) => {
+ const [changeBck,setChangeBck]=useState()
```
Comment (younessoudouh) [inline src/footer/button.jsx:5]: Initial your state! and chose good naming convention!

File: public/index.html [hunk 4]
```diff
@@ -12,6 +12,10 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&family=Rowdies:wght@300;400;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&family=Rowdies:wght@300;400;700&display=swap"
+        rel="stylesheet">
```
Comment (faouzioudouh) [inline public/index.html:18]: Why are you adding this?

File: src/footer/footer.jsx [hunk 5]
```diff
@@ -1,18 +1,30 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import "./style.css";
 import { FiSearch } from "react-icons/fi";
 import { MdOutlineMovie } from "react-icons/md";
 import { CgScreen } from "react-icons/cg";
 import { MdLocalFireDepartment } from "react-icons/md";
 import { Button } from "./button";
-
+import { Link } from "react-router-dom";
```
Comment (faouzioudouh) [inline src/footer/footer.jsx:8]: keep empty line before the function

File: src/footer/style.css [hunk 6]
```diff
@@ -8,6 +8,7 @@
     display: flex;
     justify-content: center;
     position: fixed;
+    gap: 100px;
```
Comment (faouzioudouh) [inline src/footer/style.css:11]: Why is this gap so big?

File: src/footer/style.css [hunk 7]
```diff
@@ -22,6 +23,48 @@
...
+}
+
+.ctaClick:after {
+    content: "";
+    background: #FFFFFF;
+    display: block;
+    position: absolute;
+    padding-top: 300%;
+    padding-left: 350%;
+    margin-left: -20px!important;
```
Comment (faouzioudouh) [inline src/footer/style.css:55]: Why !important?

File: src/footer/style.css [hunk 8]
```diff
@@ -22,6 +23,48 @@
...
+    margin-top: -120%;
+    opacity: 0;
+    transition: all 0.8s
+}
+
+.ctaClick:active:after {
```
Comment (faouzioudouh) [inline src/footer/style.css:61]: class name should be with good name

Comment (faouzioudouh) [reply src/footer/style.css:61]: `cta-click` not `ctaClick`

File: src/footer/button.jsx [hunk 9]
```diff
@@ -1,14 +1,18 @@
-import React, { useState } from "react";
-
+import React, { useEffect, useState } from "react";
 import "./style.css";
+
 export const Button = ({ icon, name }) => {
- const [changeBck,setChangeBck]=useState()
+  const [changeBack, setChangeBack] = useState();
+
+  const [boolean, setBoolean] = useState();
```
Comment (younessoudouh) [inline src/footer/button.jsx:7]: boolean its not a good name!

File: src/footer/style.css [hunk 10]
```diff
@@ -45,24 +45,22 @@
...
     content: "";
-    background: #FFFFFF;
+    background: #ffffff;
     display: block;
     position: absolute;
-    padding-top: 300%;
-    padding-left: 350%;
-    margin-left: -20px!important;
-    margin-top: -120%;
+    padding: 300% 350%;
```
Comment (younessoudouh) [inline src/footer/style.css:53]: px!

File: src/footer/style.css [hunk 11]
```diff
@@ -22,6 +23,45 @@
     font-family: "Roboto";
 }
 
+.cta-click {
+    position: relative;
+    border: none;
+    color: #ffffff;
+    padding: 20px;
+    width: 200px;
```
Comment (faouzioudouh) [inline src/footer/style.css:31]: Why fixed width?

File: src/footer/style.css [hunk 12]
```diff
@@ -22,6 +23,45 @@
...
+    text-align: center;
+    transition-duration: 0.4s;
+    text-decoration: none;
+    overflow: hidden;
+    cursor: pointer;
+    flex: 1;
+    padding: 6px 12px 8px;
+    max-width: 168px;
```
Comment (faouzioudouh) [inline src/footer/style.css:39]: Why max-width?

File: src/footer/style.css [hunk 13]
```diff
@@ -22,6 +23,45 @@
...
+    font-family: "Roboto";
+}
+
+.cta-click:after {
+    content: "";
+    background: #ffffff;
+    display: block;
+    position: absolute;
+    padding: 300px 250px;
+    margin: -120px -20px;
```
Comment (faouzioudouh) [inline src/footer/style.css:53]: What this?

File: src/trending.jsx [hunk 14]
```diff
@@ -1,14 +1,14 @@
...
-      return useQuery("data",()=>{
-          return fetch(`https://jsonplaceholder.typicode.com/todos`).then((res)=>res.json());
-      })
+  const useFetchData = () => {
+    return useQuery("data", () => {
+      return fetch(`https://jsonplaceholder.typicode.com/todos`).then((res) =>
+        res.json()
+      );
+    });
   };
```
Comment (faouzioudouh) [inline src/trending.jsx:10]: What is this?

**Review by faouzioudouh — APPROVED** [review]

File: src/pages/trending/movies/buttonTypeFilme/button.jsx [hunk 15]
```diff
@@ -0,0 +1,19 @@
+import React, { useEffect, useState } from "react";
+import "./styleButton.css";
```
Comment (faouzioudouh) [inline src/pages/trending/movies/buttonTypeFilme/button.jsx:2]: Keep empty line between local files and external dependencies

File: src/pages/trending/movies/buttonTypeFilme/styleButton.css [hunk 16]
```diff
@@ -0,0 +1,50 @@
...
+    align-items: center;
+    gap: 4px;
+}
+
+.cta-when-click:hover {
+    background-color: rgba(96, 109, 255, 0.87);
+}
+
+.icon-delete {
+    color: rgb(201, 203, 203);
```
Comment (faouzioudouh) [inline src/pages/trending/movies/buttonTypeFilme/styleButton.css:44]: use HEX code

File: src/pages/trending/movies/buttonTypeFilme/styleButton.css [hunk 17]
```diff
@@ -0,0 +1,50 @@
...
+    font-size: 20px;
+}
+
+.icon-delete:hover {
+    color: rgb(255, 255, 255);
```
Comment (faouzioudouh) [inline src/pages/trending/movies/buttonTypeFilme/styleButton.css:49]: use HEX code

**Pull request merged**